app/hotels/src/allHotels/__tests__/RenderSearchResults.test.js

```diff
@@ -22,44 +23,50 @@ jest.mock('../../map/allHotels/MapScreen');
 jest.mock('../AllHotelsSearchList');
 
 it('initialises correctly when it should show list', () => {
-  // $FlowExpectedError: Passing only props needed for this test
-  const wrapper = renderer.create(<RenderSearchResults {...defaultProps()} />);
+  const wrapper = renderer.create(
+    <SearchResultsContext.Provider show="list">
+      {/* $FlowExpectedError: Passing only props needed for this test */}
+      <RenderSearchResults {...defaultProps()} />
+    </SearchResultsContext.Provider>,
+  );
   expect(wrapper.root.findByProps({ testID: 'list-wrapper' }).props.style)
     .toMatchInlineSnapshot(`
-    Object {
-      "bottom": 0,
-      "end": 0,
-      "position": "absolute",
-      "start": 0,
-      "top": undefined,
-      "transform": Array [
-        Object {
-          "translateY": 0,
-        },
-      ],
-    }
-  `);
+            Object {
+              "bottom": 0,
+              "end": 0,
+              "position": "absolute",
+              "start": 0,
+              "top": undefined,
+              "transform": Array [
+                Object {
+                  "translateY": 0,
+                },
+              ],
+            }
+      `);
   expect(wrapper.root.findByProps({ testID: 'map-wrapper' }).props.style)
     .toMatchInlineSnapshot(`
-    Object {
-      "bottom": 0,
-      "end": 0,
-      "position": "absolute",
-      "start": 0,
-      "top": undefined,
-      "transform": Array [
-        Object {
-          "translateY": 1334,
-        },
-      ],
-    }
-  `);
+            Object {
+              "bottom": 0,
+              "end": 0,
+              "position": "absolute",
+              "start": 0,
+              "top": undefined,
+              "transform": Array [
+                Object {
+                  "translateY": 1334,
+                },
+              ],
+            }
+      `);
 });
 
 it('initialises correctly when it should show map', () => {
   const wrapper = renderer.create(
-    // $FlowExpectedError: Passing only props needed for this test
-    <RenderSearchResults {...defaultProps({}, 'map')} />,
+    <SearchResultsContext.Provider show="map">
+      {/* $FlowExpectedError: Passing only props needed for this test */}
+      <RenderSearchResults {...defaultProps({}, 'map')} />,
```

Comment:
Is passing `map` here necessary since you have the provider?

Reply:
True, I should remove that

Reply:
🚀